packages/complaints/lib/models/pgr_address.mapper.dart

```diff
@@ -13,6 +13,10 @@ class PgrAddressModelMapper extends SubClassMapperBase<PgrAddressModel> {
   static PgrAddressModelMapper ensureInitialized() {
     if (_instance == null) {
       MapperContainer.globals.use(_instance = PgrAddressModelMapper._());
+      EntityModelMapper.ensureInitialized().addSubMapper(_instance!);
```

Comment:
Run build runner again this should be generated. dart_mappable_builder should be from local override file. Check other mapper files as well.